.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "03:00"
+    timezone: Europe/London
+  open-pull-requests-limit: 10
```

Comment:
Do you think we could turn this down to 5, or is that too few? I personally find it a bit overwhelming when there are loads of PRs open, despite the importance of keeping dependencies up to date, what do you think?